mitodl/micromasters
# Output vars for easier debugging

#### What are the relevant tickets?
None

#### What's this PR do?
Prints out environment variables defined in `./scripts/envs.sh` for easier debugging

#### How should this be manually tested?
No changes should show up in your local environment. If you run the script directly it should show the environment variables being set.
